sys/riscv/riscv/pmap.c

```diff
@@ -2621,6 +2640,44 @@ pmap_promote_l2(pmap_t pmap, pd_entry_t *l2, vm_offset_t va,
 }
 #endif
 
+#if __has_feature(capabilities)
+static __inline uint64_t
+cheri_pte_cw(vm_page_t m, vm_prot_t prot, u_int flags)
+{
+	// Capability store and capdirty tracking using enables
```

Comment:
C-style comments, though not sure what this comment adds (and it should be before the function anyway if it's about the whole function)